server/utils/appDataVolume.ts

```diff
@@ -6,3 +6,7 @@ const DOCKER_PATH = path.join(__dirname, '../../config/DOCKER');
 export const appDataStatus = (): boolean => {
   return !existsSync(DOCKER_PATH);
 };
+
+export const appDataPath = (): string => {
+  return path.join(__dirname, '../../config');
```

Comment:
I guess an issue here is the assumption the config folder is always located at the root of the app but it can be anywhere, technically. We have an environment variable `CONFIG_DIRECTORY` to change where the config folder is located. Maybe we should check that also here?

Reply:
Oh yeah, I wasn't aware of the env var, but that sounds great.  I'll look into this tomorrow or Saturday 😺

Reply:
Okay nevermind, my brain wouldn't let me leave this alone for so long.  Fixed!  Also found another location that wasn't using `CONFIG_DIRECTORY` so I fixed that as well.